project/common.scala

```diff
@@ -21,14 +21,14 @@ object Common {
   def dispatchOAuthDep =
     "net.databinder" %% "dispatch-oauth" % dispatchVersion
 
-  def integrationTestDeps(sv: String) = (specs2Dep(sv) :: dispatchDeps) map { _ % "test" }
+  def integrationTestDeps(sv: String) = (specs2Dep(sv) :: Nil) map { _ % "test" }
 
   val settings: Seq[Setting[_]] = Defaults.coreDefaultSettings ++ Seq(
     organization := "net.databinder",
 
-    version := "0.8.4",
+    version := "0.8.4-RC1",
```

Comment:
Building a Scala 2.12 for 0.8.4 should definitely be a release candidate, for now, since the test suite can't be run in Scala 2.12.  A 0.8.4-RC1 will still be useful.